tests/test_main.py

```diff
@@ -522,54 +732,112 @@ def check_filename(self, filename, check_names):
-                    # Assignment to attributes and subscripts is less magical
-                    # but can also fail fairly easily, so we can't guarantee
-                    # that every node can be identified with some instruction.
-                    continue
+                # skip some cases cases 
+                if sys.version_info < (3, 11):
+                    if is_unary_not(node):
+                        continue
 
-                if isinstance(ctx, (ast.Del, getattr(ast, 'Param', ()))):
-                    assert not values
-                    continue
+                    if sys.version_info >= (3, 8):
+                        if is_annotation(node):
+                            continue
 
-                if isinstance(node, ast.Compare):
-                    if sys.version_info >= (3, 10):
+                    ctx = getattr(node, 'ctx', None)
+                    if isinstance(ctx, ast.Store):
+                        # Assignment to attributes and subscripts is less magical
+                        # but can also fail fairly easily, so we can't guarantee
+                        # that every node can be identified with some instruction.
                         continue
-                    if len(node.ops) > 1:
-                        assert not values
+
+                    if isinstance(ctx, (ast.Del, getattr(ast, 'Param', ()))):
+                        assert not values, [ast.dump(node), values]
                         continue
 
-                    if is_unary_not(node.parent) and isinstance(node.ops[0], (ast.In, ast.Is)):
+                    if isinstance(node, ast.Compare):
+                        if sys.version_info >= (3, 10):
+                            continue
+                        if len(node.ops) > 1:
+                            assert not values
+                            continue
+
+                        if is_unary_not(node.parent) and isinstance(
+                            node.ops[0], (ast.In, ast.Is)
+                        ):
+                            continue
+
+                    if is_literal(node):
+                        continue
+                else:
+                    if is_unary_not(node):
                         continue
 
-                if is_literal(node):
-                    continue
+                    if is_annotation(node):
+                        continue
+
+
+                    # (is/is not) None
+                    if (
+                        isinstance(node, ast.Compare)
+                        and len(node.ops) == 1
+                        and isinstance(node.ops[0], (ast.IsNot, ast.Is))
+                        and len(node.comparators) == 1
+                        and isinstance(node.comparators[0], ast.Constant)
+                        and node.comparators[0].value == None
+                    ):
+                        continue
+
+                    if is_literal(node) and not isinstance(node,ast.Constant):
```

Comment:
The current testing code `check_code` checks not all bytecodes. `BUILD_LIST` does not get checked. I extended this list a bit but not to every bytecode type (could be done later if necessary). 
Are there use cases where a user of `executing` might wants to map this bytecode for example?
There is no python code executed inside `BUILD_LIST` which could call `executing` with a `frame`
A script to show what gets mapped and what not can be found [here](https://github.com/15r10nk/executing/blob/analyse/analyse.py).

And yes, I think the code does something wrong, will check it further.

Reply:
Sorry, I forget sometimes that it's not enough for instruction positions to match a node, it has to be verified as a known case. Your verifier is much stricter and more extensive than I was expecting and I'm still getting used to it. It's fine to not identify things like BUILD_LIST.

> And yes, I think the code does something wrong,

Because LOAD_CONST isn't checked either, right?